controllers/configurationpolicy_controller.go

```diff
@@ -1970,30 +2007,35 @@ func getNamesOfKind(
 	dclient dynamic.Interface,
 	complianceType string,
 	zeroValueEqualsNil bool,
-) (kindNameList []string) {
+) (kindNameList []string, allResourceList []string) {
```

Comment:
Refactored

Reply:
Could you please update the documentation comment of `getNamesOfKind` to indicate what `allResourceList` is?